plane/fit_plane.py

```diff
@@ -177,7 +177,20 @@ def get_plane_equation(self):
         d = -np.dot(normal_vec, self.h)
         
         return a,b,c,d
-        
+
+    def check_uv_angle(self):
+        cos_theta = np.dot(self.u, self.v) / (self.u_norm_mm() * self.v_norm_mm())
+        # Angle (radians)
+        theta_rad = np.arccos(cos_theta)
```

Comment:
No need to take the cos.

We can just compare
np.dot(self.u_direction(),self.v_direction()) > 0.025